### PR TITLE
Dungeon Crawl Huge Sprite Update

### PR DESCRIPTION
**Pull request by PolishUser**
This PR adds:

1. New Sprites
2. A foundation for weapons
3. A proc which can be used to give and equip items on mobs
4. A couple of concepts which are open for discussion/modification
5. Last but not least fixes corrupted .dmi files located in _equipable_

### CODE REVIEW

File: code/mobs/creatures/undead.dm [hunk 1]
```diff
@@ -14,14 +14,29 @@
...
 	icon_state = "zombie"
 
-/mob/undead/skeleton_warrior
+/mob/undead/skeleton/skeleton_warrior
 	name = "skeleton warrior"
 	desc = "Worse to the bone."
 	max_hp = 20
-	icon_state = "skeleton_warrior"
+
+/mob/undead/skeleton/skeleton_warrior/New()
```
Comment (TheChosenEvilOne) [inline code/mobs/creatures/undead.dm]: should probably be just `skeleton/warrior`

File: code/mobs/mob.dm [hunk 2]
```diff
@@ -112,6 +113,12 @@
 	else
 		return FALSE
 
+/mob/proc/equip_new(path)
+	var/I = new path
+	pick_up(I)
+	equip(I)
+	update_stats()
```
Comment (TheChosenEvilOne) [inline code/mobs/mob.dm]: `equip` calls `update_stats`, this shouldn't be necessary.

File: code/mobs/creatures/arachnid.dm [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+	name = "Baby Spider"
+	desc = "AAAAAAA!"
+	max_hp = 3
+	icon_state = "baby_spider"
+
+/mob/spider/queen
+	name = "Spider Queen"
+	desc = "She's huge!"
+	max_hp = 200
+	icon = 'icons/mobs/32x32.dmi' // Will it work 32x32???
```
Comment (TheChosenEvilOne) [inline code/mobs/creatures/arachnid.dm]: as discussed before, likely not going to have anything multi-tile.

File: code/mobs/mob.dm (hunk 2 above)
Comment (PolishUser) [reply code/mobs/mob.dm]: Unfortunately, if I do not add _update_stats_, the health values will become nil.

File: code/mobs/creatures/arachnid.dm (hunk 3 above)
Comment (PolishUser) [reply code/mobs/creatures/arachnid.dm]: I guess I'll have to rethink this.

File: code/mobs/mob.dm (hunk 2 above)
Comment (TheChosenEvilOne) [reply code/mobs/mob.dm]: that means there is a bug elsewhere and that has to be fixed. You shouldn't add hacks to hide bugs like this.

Comment (PolishUser) [reply code/mobs/mob.dm]: Ok, fixed

**Pull request merged**